engine/src/flutter/lib/web_ui/lib/src/engine/skwasm/skwasm_impl/picture.dart

```diff
@@ -49,6 +49,14 @@ class SkwasmPicture extends SkwasmObjectWrapper<RawPicture> implements LayerPict
     pictureRef(handle);
     return SkwasmPicture.fromHandle(handle, isClone: true);
   }
+
+  @override
+  String toString() {
+    return 'SkwasmPicture(${handle.address})';
+  }
+
+  @override
+  bool get isDisposed => debugDisposed;
```

Comment:
If `debugDisposed` is now meant to be used in production code, maybe let's rename it to `_isDisposed` and use the new `isDisposed` in tests?

Reply:
I wanted to do that initially but `debugDisposed` is part of the public `dart:ui` API: https://api.flutter.dev/flutter/dart-ui/Picture-class.html

So, I figured we could keep `debugDisposed` as the "check if this is disposed but throw if not in debug mode" and have `isDisposed` (on the `LayerPicture` interface) be an engine-only "check if this is disposed"